plutus-ledger/plutus-ledger.cabal

```diff
@@ -80,7 +80,7 @@ library
     build-depends:
         base >=4.9 && <5,
         aeson -any,
-        base16-bytestring -any,
+        base16-bytestring <1, 
```

Comment:
Is this related to the rest of this PR?

Reply:
only in that it is required when removing the deployment server as that haskell component was forcing this restriction to be the case.